presto-hive/src/test/java/com/facebook/presto/hive/parquet/AbstractTestParquetReader.java

```diff
@@ -1559,6 +1563,16 @@ public void testStructMaxReadBytes()
                 maxReadBlockSize);
     }
 
+    @Test
+    public void testNullableNullCount()
+    {
+        PrimitiveType primitiveType = new PrimitiveType(OPTIONAL, BINARY, "Test column");
+        org.apache.parquet.format.Statistics statistics = new Statistics();
```

Comment:
do we need `org.apache.parquet.format.Statistics`? or just `Statistics` is good?

Reply:
done. thank you : )